src/OMSimulatorLib/DummyTLM/TLMInterface.h

```diff
@@ -31,7 +31,7 @@
 
 #ifndef _DUMMY_TLM_INTERFACE_H_
 #define _DUMMY_TLM_INTERFACE_H_
-#define NO_TLM 1
+//#define NO_TLM 1
```

Comment:
Why is it commented out?

Reply:
My bad. I only commented it out to temporary force enable TLM in my build environment.